docs/docs/guides/architecture-scenarios/b2c.mdx

```diff
@@ -0,0 +1,113 @@
+---
+title: B2C
+---
+
+import Column from '../../../src/components/column';
+
+## Business to Customer
+
+Users in general come with different needs. You may have end users, employees, or even customers from other parties (B2B). 
+This groups of users usually don't share the same intentions and use applications differently.
+When planning your applications, investing time in researching your apps architecture is vital for later feature upgrades and enhancements as those changes come in with heftier price points if you have to make bigger changes.
+
+This guide introduces you to the grouping and structuring of ZITADEL projects which forms the base for all projects. This can be used as a quick start to the [B2B scenario](./b2b), which is merely focused on planning considerations if you are having projects with multiple organizations.
+
+The journey of this guide starts with creating an Organization, the outermost layer of ZITADEL, as it is the vessel for projects, roles, applications and users.
+Creation can be done from [ZITADEL Console](https://console.zitadel.ch/org/create). You can choose your current account for the organization owner or create a new one.
+
+Depending on your Software Development Life Cycle (SDLC) you will have to create multiple organizations to keep your data from different environments separate.
+
+| Environment | Organization Name     | Description             |
+| :---------- | :-------------------- | ----------------------- |
+| Development | **acme-dev**          | The development environment and organization |
+| Testing     | **acme-test**         | An environment for testing and production preview |
+| Production  | **acme-prod**         | The production environment and organization |
```

Comment:
Do we really mention this as "best practice" scenario? In my opinion we recommended using different projects for different environments? also this would not match with our new pricing, because b2c only gets 2 organizations

Reply:
Then a dev user (fake account) could be used in production? would we want that? 🤔

Reply:
I would always use the outermost layer possible, which is an organization.

Reply:
@fforootd what are your thoughts?

Reply:
@fgerschwiler 
>  also this would not match with our new pricing, because b2c only gets 2 organizations

In that case it might be a good nudge 😁 

@peintnermax 
> I would always use the outermost layer possible, which is an organization.
Well this is true to some degree, but for now i would say this is out of scope. Actually both ways can work and I think we should leave this open for customers choice 😄